src/store/index.js

```diff
@@ -0,0 +1,11 @@
+import { createStore, applyMiddleware } from "redux";
+import thunk from "redux-thunk";
+import rootReducer from "./reducers/rootReducer";
+
+export default function store(initialState={}) {
```

Comment:
What do you think about the naming of this function? It creates a store but it's named store. Can we have a better name or?